webview/src/test/dragDrop.ts

```diff
@@ -50,7 +50,7 @@ export const dragEnter = (
   })
 
   act(() => {
-    jest.advanceTimersByTime(1)
+    jest.advanceTimersByTime(501)
```

Comment:
[Q] Why is this needed?

Reply:
It isn't. I added `dragLeave` in there to test something and the `500` was needed then. Turns out that wasn't how to solve the test and I forgot to revert this.